src/mempool.hpp

```diff
@@ -291,12 +306,18 @@ namespace PYGPU_PACKAGE
         free_held();
       }
 
-      unsigned active_blocks()
+      size_type active_blocks()
```

Comment:
Could probably mark these methods `const`, but not very important.

Reply:
eba3dcc